hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -4543,4 +4543,141 @@
       allowing for better identification and analysis of performance issues.
     </description>
   </property>
+
+  <property>
+    <name>hdds.datanode.dns.interface</name>
+    <value>default</value>
+    <tag>OZONE, DATANODE</tag>
+    <description>
+      The name of the Network Interface from which a data node should
+      report its IP address. e.g. eth2. This setting may be required for some
+      multi-homed nodes where the DataNodes are assigned multiple hostnames
+      and it is desirable for the DataNodes to use a non-default hostname.
+    </description>
+  </property>
+  <property>
+    <name>hdds.datanode.dns.nameserver</name>
+    <value>default</value>
+    <tag>OZONE, DATANODE</tag>
+    <description>
+      The host name or IP address of the name server (DNS) which a DataNode
+      should use to determine its own host name.
+    </description>
+  </property>
+  <property>
+    <name>hdds.datanode.hostname</name>
+    <value></value>
+    <tag>OZONE, DATANODE</tag>
+    <description>
+      Optional. The hostname for the Datanode containing this
+      configuration file.  Will be different for each machine.
+      Defaults to current hostname.
+    </description>
+  </property>
+  <property>
+    <name>hdds.datanode.data.dir</name>
```

Comment:
Is this an alias for `hdds.datanode.dir`? Looks like we should keep one and deprecate the other. I think `hdds.datanode.dir` is the most commonly used one.